atomic_reactor/utils/yum.py

```diff
@@ -50,7 +49,7 @@ def filename(self):
         if not basename.endswith(REPO_SUFFIX):
             basename += REPO_SUFFIX
         if self.add_hash:
-            suffix = '-' + md5(self.repourl.encode('utf-8')).hexdigest()[:5]  # nosec
+            suffix = '-' + sha256sum(self.repourl, abbrev_len=5)
```

Comment:
change also docstring reflecting the change

Reply:
Updated. PTAL.